python-input-file.yaml

```
@@ -0,0 +1,44 @@
+id: python-input-file
+namespace: company.team
+
+tasks:
+  - id: download_file
+    type: io.kestra.plugin.core.http.Download
+    uri: https://huggingface.co/datasets/kestra/datasets/raw/main/csv/orders.csv
+
+  - id: get_total_rows
+    type: io.kestra.plugin.scripts.python.Script
+    beforeCommands:
+      - pip install pandas
+    inputFiles:
+      input.csv: "{{ outputs.download_file.uri }}"
+    script: |
+      import pandas as pd
+
+      # Path to your CSV file
+      csv_file_path = "{{ workingDir }}/input.csv"
```

We shouldn't need this because we're using inputFiles. Also, I want to keep the Python code as vanilla as possible

```suggestion
      csv_file_path = "input.csv"
```